### ISSUE TITLE
ruboto setup -y - FATAL -- : undefined method `slice' for nil:NilClass

### ISSUE THREAD

**Issue by andrewhavens** [ruboto/ruboto#717]
I am trying out Ruboto for the first time, based on the instructions in this blog post: http://ruboto.org/news/2015/04/05/Ruboto-1.3.0-release-doc.html.

```bash
$ ruby -v
ruby 2.1.2p95 (2014-05-08 revision 45877) [x86_64-darwin13.0]

$ gem install ruboto
5 gems installed

$ ruboto setup -y
WARN: Unresolved specs during Gem::Specification.reset:
      rake (~> 10.0)
      rubyzip (~> 1.0)
WARN: Clearing out unresolved specs.
Please report a bug if this causes problems.
Java runtime             : Found
Java Compiler            : Found
Apache ANT               : Not found
Android Package Installer: Not found
Android Emulator         : Not found
F, [2015-04-08T12:25:18.274112 #53333] FATAL -- : undefined method `slice' for nil:NilClass (NoMethodError)
/Users/andrew/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/ruboto-1.3.0/lib/ruboto/util/setup.rb:200:in `check_for_haxm'
/Users/andrew/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/ruboto-1.3.0/lib/ruboto/util/setup.rb:168:in `check_all'
/Users/andrew/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/ruboto-1.3.0/lib/ruboto/util/setup.rb:23:in `setup_ruboto'
/Users/andrew/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/ruboto-1.3.0/lib/ruboto/commands/base.rb:453:in `run'
/Users/andrew/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/main-5.3.2/lib/main/program/class_methods.rb:155:in `block in run'
/Users/andrew/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/main-5.3.2/lib/main/program/class_methods.rb:144:in `catch'
/Users/andrew/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/main-5.3.2/lib/main/program/class_methods.rb:144:in `run'
/Users/andrew/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/main-5.3.2/lib/main/factories.rb:18:in `run'
/Users/andrew/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/main-5.3.2/lib/main/factories.rb:25:in `Main'
/Users/andrew/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/ruboto-1.3.0/lib/ruboto/commands/base.rb:28:in `main'
/Users/andrew/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/ruboto-1.3.0/bin/ruboto:11:in `<top (required)>'
/Users/andrew/.rbenv/versions/2.1.2/bin/ruboto:23:in `load'
/Users/andrew/.rbenv/versions/2.1.2/bin/ruboto:23:in `<main>'
```

**Comment by donv** [comment]
Hi @andrewhavens !

Thanks for the report.  Looks like we have a file name mismatch for the HAXM setup.

I am really swamped this week, so I will probably not have time to look at it before the weekend.

Feel free to look at the code and send a pull request, and I will merge it and make a new release.

**Comment by NaN1488** [comment]
@donv @andrewhavens 

I fixed it. Just replace this line https://github.com/ruboto/ruboto/blob/1.3.0/lib/ruboto/util/setup.rb#L200

with this:
```ruby 
@haxm_installer_version = (@haxm_installer_loc.nil? nil : @haxm_installer_loc.slice(/IntelHAXM_1.1.1_/)[10..-2])
```

Not elegant but at least I have this output
```
NaN:ruboto nan$ bundle exec ruboto setup
Java runtime             : Found
Java Compiler            : Found
Apache ANT               : Found
Android Package Installer: Found
Android Emulator         : Found
Intel HAXM               : Found
Android SDK Command adb  : Found
Android SDK Command dx   : Found
Platform SDK android-16  : Found

    *** Ruboto setup is OK! ***
```
I let the pull request to you guys

**Comment by danielpassos** [comment]
@andrewhavens @NaN1488 Could you confirm if it was fixed in https://github.com/ruboto/ruboto/pull/728?

**Comment by donv** [comment]
@daneb Is this something you have covered in your changes?

**Comment by donv** [comment]
OK, I'll assume that this was fixed in #728.
